subsys/net/lib/azure_iot_hub/src/azure_iot_hub.c

```diff
@@ -849,6 +849,9 @@ static int client_broker_init(struct mqtt_client *const client, bool dps)
 	tls_cfg->sec_tag_count		= ARRAY_SIZE(sec_tag_list);
 	tls_cfg->sec_tag_list		= sec_tag_list;
 	tls_cfg->session_cache		= TLS_SESSION_CACHE_DISABLED;
+#if defined(CONFIG_AZURE_IOT_HUB_NATIVE_TLS)
+	tls_cfg->set_native_tls		= CONFIG_AZURE_IOT_HUB_NATIVE_TLS;
```

Comment:
I guess this could be simplified a bit to save a few lines and an ifdef:
```
tls_cfg->set_native_tls		= IS_ENABLED(CONFIG_AZURE_IOT_HUB_NATIVE_TLS);
```